packages/babel-preset-stage-0/index.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  presets: [
+    require("babel-preset-stage-1")
```

Comment:
Stage 0 shouldn't include stage 1. Stage 0 is the earliest while stage 4 is the latest. You've got these the wrong way around.